src/Signature/SignatureV4.php

```diff
@@ -14,21 +14,27 @@ class SignatureV4 implements SignatureInterface
 {
     use SignatureTrait;
     const ISO8601_BASIC = 'Ymd\THis\Z';
+    const UNSIGNED_PAYLOAD = 'UNSIGNED-PAYLOAD';
```

Comment:
This constant is also available on [`S3SignatureV4`](https://github.com/aws/aws-sdk-php/blob/master/src/Signature/S3SignatureV4.php#L12), which extends this class. You could remove the constant from S3SignatureV4.